src/QuDirac.jl

```diff
@@ -1,12 +1,12 @@
 module QuDirac
-    
+
     using Compat
     using Iterators.product
 
-    if !(v"0.3-" <= VERSION < v"0.4-")
-        warn("QuDirac v0.1 only officially supports the v0.3 release of Julia. Your version of Julia is $VERSION.")
+    if !(VERSION < v"0.6")
+        warn("QuDirac v0.1 only officially supports the v0.6 release of Julia. Your version of Julia is $VERSION.")
     end
-    
```

Comment:
This whole warning should probably go away, any version handling should just go into the REQUIRE anyway